### DISCUSSION TITLE
Cannot create FMOD - error during batch synthesis

### DISCUSSION THREAD

**Discussion by nehvaleem** [RageAgainstThePixel/com.rest.elevenlabs#82] [Q&A]
# Bug Report

## Overview
I am creating synthesized voices in batch, and some of the results aren't properly imported by unity. I am getting the exception `Error: Cannot create FMOD::Sound instance for clip "" (FMOD error: Error loading file. )` (stacktrace is attached at the end). I am using the latest 3.2.9 version.

The issue is that for example, VLC is able to play the file. And I can reconvert it using some other 3rd party tool, but it doesn't make too much sense in the batch processing context.

## Additional context

```csharp
Error: Cannot create FMOD::Sound instance for clip "" (FMOD error: Error loading file. )
UnityEngine.StackTraceUtility:ExtractStackTrace ()
Utilities.WebRequestRest.Rest/<DownloadAudioClipAsync>d__43:MoveNext () (at ./Library/PackageCache/com.utilities.rest@2.5.3/Runtime/Rest.cs:680)
System.Runtime.CompilerServices.AsyncMethodBuilderCore/MoveNextRunner:Run ()
Utilities.Async.AwaiterExtensions:RunOnUnityScheduler (System.Action) (at ./Library/PackageCache/com.utilities.async@2.1.3/Runtime/Async/AwaiterExtensions.cs:248)
Utilities.Async.SimpleCoroutineAwaiter:Complete (System.Exception) (at ./Library/PackageCache/com.utilities.async@2.1.3/Runtime/Async/SimpleCoroutineAwaiter.cs:47)
Utilities.Async.AwaiterExtensions/<ReturnVoid>d__27:MoveNext () (at ./Library/PackageCache/com.utilities.async@2.1.3/Runtime/Async/AwaiterExtensions.cs:334)
Unity.EditorCoroutines.Editor.EditorCoroutine:ProcessIEnumeratorRecursive (System.Collections.IEnumerator) (at ./Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:148)
Unity.EditorCoroutines.Editor.EditorCoroutine:MoveNext () (at ./Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:115)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()
```

**Comment by StephenHodgson** [comment]
How did you create this file?

Were you streaming or using the normal endpoint?

**Comment by nehvaleem** [comment]
> How did you create this file?
> 
> Were you streaming or using the normal endpoint?

Using TextToSpeechEndpoint. I am creating 50-100 files in batch and around 5% of them aren't importing correctly to unity :(

**Comment by StephenHodgson** [comment]
What format are they generated for?

**Comment by StephenHodgson** [comment]
FWIW I also use batch processing and have not ran into this. It'd be helpful if you shared your implementation.
